[2021-07-05 10:23:09,595] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: get_as_dataframe.getting_data.get_cataract_sex 2021-07-04T00:00:00+00:00 [queued]>
[2021-07-05 10:23:09,633] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: get_as_dataframe.getting_data.get_cataract_sex 2021-07-04T00:00:00+00:00 [queued]>
[2021-07-05 10:23:09,633] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-05 10:23:09,633] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-05 10:23:09,635] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-05 10:23:09,662] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): getting_data.get_cataract_sex> on 2021-07-04T00:00:00+00:00
[2021-07-05 10:23:09,667] {standard_task_runner.py:52} INFO - Started process 17427 to run task
[2021-07-05 10:23:09,681] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'get_as_dataframe', 'getting_data.get_cataract_sex', '2021-07-04T00:00:00+00:00', '--job-id', '1143', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/get_as_Dataframe.py', '--cfg-path', '/tmp/tmpas5t_pl2', '--error-file', '/tmp/tmpgd3gblmd']
[2021-07-05 10:23:09,683] {standard_task_runner.py:77} INFO - Job 1143: Subtask getting_data.get_cataract_sex
[2021-07-05 10:23:09,812] {logging_mixin.py:104} INFO - Running <TaskInstance: get_as_dataframe.getting_data.get_cataract_sex 2021-07-04T00:00:00+00:00 [running]> on host airflowvm
[2021-07-05 10:23:10,039] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_as_dataframe
AIRFLOW_CTX_TASK_ID=getting_data.get_cataract_sex
AIRFLOW_CTX_EXECUTION_DATE=2021-07-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-04T00:00:00+00:00
[2021-07-05 10:23:10,074] {base.py:69} INFO - Using connection to: id: ***_default. Host: localhost, Port: 5432, Schema: , Login: ***, Password: ***, extra: {'cursor': 'realdictcursor'}
[2021-07-05 10:23:10,142] {python.py:151} INFO - Done. Returned value was: {"Cataract Females":{"0":"(2593)"},"Cataract Males":{"0":"(1871)"}}
[2021-07-05 10:23:10,285] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=get_as_dataframe, task_id=getting_data.get_cataract_sex, execution_date=20210704T000000, start_date=20210705T102309, end_date=20210705T102310
[2021-07-05 10:23:10,416] {dagrun.py:495} WARNING - Failed to get task '<TaskInstance: get_as_dataframe.get_diseases 2021-07-04 00:00:00+00:00 [removed]>' for dag 'get_as_dataframe'. Marking it as removed.
[2021-07-05 10:23:10,424] {dagrun.py:495} WARNING - Failed to get task '<TaskInstance: get_as_dataframe.diseases_treatment 2021-07-04 00:00:00+00:00 [removed]>' for dag 'get_as_dataframe'. Marking it as removed.
[2021-07-05 10:23:10,426] {dagrun.py:495} WARNING - Failed to get task '<TaskInstance: get_as_dataframe.write_diseases 2021-07-04 00:00:00+00:00 [removed]>' for dag 'get_as_dataframe'. Marking it as removed.
[2021-07-05 10:23:10,467] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-05 10:23:10,504] {local_task_job.py:151} INFO - Task exited with return code 0
